doc/life_of_a_language_feature.md

```diff
@@ -1,37 +1,43 @@
-# Dart language change process
+# Dart language evolution process
 
 ## User issue or feature request
 
 Features and changes arise from perceived user issues or feature requests.
-Feature requests should be filed in
-the [language repo](http://github.com/dart-lang/language/issues/).  We may close issues
-that we believe that we will not adress for whatever reason, or we may keep
-issues open indefinitely if we believe they are something that we may wish to
-address in the future.  Feature request/problem issues are primarily for
-documentation of the user problem to be solved.
+Feature requests should be filed in the [language
+repo](https://github.com/dart-lang/language/issues/new?labels=language-request),
```

Comment:
Are there any other kind of "request" than "language-request"? Otherwise the "language" is redundant, and the label should just be "request".

I assume this matches the current "problem" label. I deliberately split the labels into "problem" and "solution". A feature request *too often* is just a "solution" without a statement of the problem it solves. That makes it very hard to discuss the adequacy and generality of the feature. If the problem is discussed, then we might recognize multiple problems that can be solved by a single solution.